python/doc/pyplots/ConditionalRandomVector.py

```diff
@@ -5,8 +5,8 @@
 ot.RandomGenerator.SetSeed(0)
```

Comment:
this file should be moved to DeconditionedRandomVector.py now that I merged #2930